index.html

```diff
@@ -4,30 +4,73 @@
     <title>Startup Matchmaker</title>
     <link href="http://fonts.googleapis.com/css?family=Source+Sans+Pro:400,700|Oswald:700|Merriweather:700,700italic" rel="stylesheet" type="text/css">
     <link rel="stylesheet" type="text/css" href="css/main.css">
+    <link rel="stylesheet" type="text/css" href="css/style.css">
   </head>
   <body>
-    Startup Matchmaker
+   <header> 
+   <div class="header">
+   <h1>Startup Matchmaker</h1>
+   <nav class="clearfix">
+    <a href="#" id="menu">&#9776;</a>
+    <a href="#">Developers </a>
+    <a href="#">Designers </a>
+    <a href="#">How it Works </a>
+    <a href="#">Our Team </a>
+    <a href="#">Blog </a>
 
-    Developers Designers How it Works Our Team Blog
+    </nav>
+    </div>
+    </header>
 
-    Because two brains taste are better than one.
+   <blockquote>
+   <h2>Because two brains <span style="text-decoration:line-through; color: gray;">taste</span><span style="text-decoration:underline;text-decoration-color: yellow;"> are</span> better than one.</h2>
+   </blockquote>
 
-    Meet your Match!
-    Have a great idea for a product, but need help making it a reality? We"re here to help. Startup Matchmaker is the best place for designers and developers to find each other.
-    Create a Profile
-   
-    Create a Profile
-    Are you a Designer? Developer? Put yourself out there so that others can find you!
-    Sign Up Now
     
-    Find a Developer
-    Looking for a fantastic developer to work with on the next big thing? Look no further.
-    Start Your Search
+    <section class="image">
+    <div id="clearfix">
+    <div id="wrapper">
+    <h3>Meet your Match!</h3>
+    <p>Have a great idea for a product, but need help making it a reality? We"re here to help. Startup Matchmaker is the best place for designers and developers to find each other.</p>
+    <button class="createp"><a href="#">Create a Profile</a>
```

Comment:
@vdigregorio no need for the button tag. Only use <a> tags for all the buttons on this page.